operations/iuf/workflows/management_rollout.md

```diff
@@ -21,6 +21,11 @@ Once this step has completed:
 
 - Host firmware has been updated on management nodes
 
+Note: In CSM 1.6, all the worker nodes are configured as iSCSI SBPS targets. In CSM 1.7.0, selective worker node
+personalization is introduced where if user/admin wants to limit some worker nodes to be configured as iSCSI targets,
+then it requires to create an HSM group by name `iscsi_worker` and add the worker node xnames to this group which are
+intended to be configured as iSCSI targets. For details, see [Management Node Personalization](../configuration_management/Management_Node_Personalization.md) and [worker node personalization](../iscsi_sbps/iscsi_sbps.md#1-worker-node-personalization)
```

Comment:
These needs to be relative links. But I'm not sure these are the right 2 links. I don't think it makes sense to link to the main Management Node Personalization page, because that doesn't talk about iSCSI at all.

I think this should be something like:
```
Before proceeding with the upgrade, administrators are encouraged to learn more about this new feature in
<link to overview of new feature>. Administrators who do not wish to use this new feature can proceed with the
upgrade. For administrators who do wish to use this new feature, see <link to upgrade-specific procedure for enabling this> for the steps to be taken before proceeding with the upgrade.